cli/cmd/clean.go

```diff
@@ -120,15 +118,22 @@ func internalCleanModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		status := running.Status(&run)
 		if status.Code == process_utils.ProcessStoppedCode {
 			var statusMsg string
+			var postfix string
+
+			if cmdCtx.Cli.Verbose {
+				postfix = "\n"
+			}
 
-			err := clean(&run)
+			filesAmount, err := clean(&run)
 			if err != nil {
-				statusMsg = "[ERR] " + err.Error()
+				statusMsg = "[ERR] " + err.Error() + postfix
+			} else if filesAmount == 0 {
+				statusMsg = "[SKIPPED]" + postfix
```

Comment:
Is this status helpful? I think if there is nothing to remove, the status can be OK.